src/main/java/com/softserve/academy/spaced/repetition/controller/AccountController.java

```diff
@@ -23,27 +23,27 @@
 public class AccountController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AccountController.class);
-    private final AccountService accountService;
+    private final AccountServiceImpl accountServiceImpl;
```

Comment:
the name of variable should stay the same `accountService`

Reply:
Okey, one sec